### PR TITLE
feat(cli): Added new info command

### LINKED ISSUE #812: Having a dedicated command to show up local environment information
### Use case

What about including a dedicated command such as `preact info` which shows up necessary local environment information. With that, all the user has to do is to copy and paste the relevant information useful especially while submitting bug reports.

`preact info`

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
feature

**Did you add tests for your changes?**
Nope
**Summary**
prints debugging information about the local environment

`preact info`

```sh
Environment Info:

  System:
    OS: Linux 4.18 Ubuntu 18.10 (Cosmic Cuttlefish)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  Binaries:
    Node: 8.11.4 - /usr/bin/node
    Yarn: 1.17.0-0 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 74.0.3729.157
    Firefox: 66.0.5
  npmGlobalPackages:
    preact-cli: 3.0.0-rc.2
```

Closes #812 

**Does this PR introduce a breaking change?**
Nope

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @lukeed

**Comment by lukeed** [comment]
Would we prefer to customize the output of `-v, --version` flag? I can allow that fairly easily within sade... it feels a little odd to have a separate command for this, not sure why

**Comment by jamesgeorge007** [comment]
@lukeed Most scaffolding tools come shipped with an `info` command these days and I think there is no harm in following the similar approach here :thinking: 
Having it along the `version` flag would be overkill IMO.

**Comment by lukeed** [comment]
I've not seen that tbh. Customizing `--version` output would be _instead of_ adding a new `info` command. It would perhaps look something like this:

```
$ preact --version

  Preact:
    cli: 3.0.0-rc.2
    core: 8.4.2
    compat: 3.19.0
    router: 2.6.1
  System:
    OS: Linux 4.18 Ubuntu 18.10 (Cosmic Cuttlefish)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  Binaries:
    Node: 8.11.4 - /usr/bin/node
    Yarn: 1.17.0-0 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 74.0.3729.157
    Firefox: 66.0.5
```

I have no strong opinions either way. Just mentioning that the possibility is there if we want it – since `--version` is already there anyway

**Comment by jamesgeorge007** [comment]
Have a look at some of the utilities with a dedicated `info` command : [envinfo-integration](https://github.com/tabrindle/envinfo#integration)

**Comment by jamesgeorge007** [comment]
Also, the `version` switch is dedicated to show up version information specifically relating to the tooling as implemented by the argument parsers out there. What I would say is to have a dedicated command for the purpose.

**Comment by ForsakenHarmony** [comment]
Yeah, dedicated command is fine imo

**Comment by lukeed** [comment]
Cool, okay 👍 just wanted to offer another option just in case

**Comment by jamesgeorge007** [comment]
@lukeed Is the current implementation fine? 
Are we all set to ship it :thinking:

File: packages/cli/lib/index.js [hunk 1]
```diff
@@ -92,4 +93,20 @@ prog
...
+	.describe('Print out debugging information about the local environment')
+	.action(() => {
+		console.log();
+		console.log('Environment Info:');
+		envinfo
+			.run({
+				System: ['OS', 'CPU'],
+				Binaries: ['Node', 'Yarn', 'npm'],
+				Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+				npmGlobalPackages: ['preact-cli'],
```
Comment (ForsakenHarmony) [inline packages/cli/lib/index.js]: good to merge if you also add the local versions of `preact`, `preact-compat` and `preact-cli` (myb `preact-router` and `preact-render-to-string`

Comment (jamesgeorge007) [reply packages/cli/lib/index.js]: :+1:

**Comment by jamesgeorge007** [comment]
cc @ForsakenHarmony

**Comment by ForsakenHarmony** [comment]
Oh one last thing, could you add it to the issue template (and readme)? 👀

**Comment by jamesgeorge007** [comment]
cc @ForsakenHarmony

**Comment by ForsakenHarmony** [comment]
🎉

**Pull request merged**